docs/docs/advanced/time-based-logic.md

```diff
@@ -2,7 +2,7 @@
 sidebar_position: 61
 ---
 
-# Time-Based Logic
+# Time-Based Games
```

Comment:
Rename the file to match?

Also, what about "Using Time in your Games"? "Time-Based Games" sounds like it only refers to games that rely heavily on timers or won't work without them, whereas they could only be a minor addition not affecting gameplay

Reply:
Good idea, changed in https://github.com/rune/rune-games-sdk/pull/175/commits/bb268dc290275a4f7726d1601b2539ee0411b050!